src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/DelayLoadHelperImport.cs

```diff
@@ -67,5 +67,17 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
                 new DependencyListEntry(ImportSignature, "Signature for ready-to-run fixup import"),
             };
         }
+
+        public override int CompareToImpl(ISortableNode other, CompilerComparer comparer)
+        {
+            DelayLoadHelperImport otherNode = (DelayLoadHelperImport)other;
+            int result = _useVirtualCall.CompareTo(otherNode._useVirtualCall);
+            if (result != 0) return result;
```

Comment:
Please use a new line here. Same for all other places.

Reply:
Where do you mean?

Reply:
I mean:
``` c#
if (result != 0)
    return result;
```